projects/core/components/textfield/textfield.component.ts

```diff
@@ -180,4 +181,15 @@ export class TuiTextfieldComponent<T> implements TuiDataListHost<T>, AfterConten
     protected onResize({contentRect}: ResizeObserverEntry): void {
         this.el.style.setProperty('--t-side', tuiPx(contentRect.width));
     }
+
+    protected onClick(event: Event): void {
+        if (
+            this.el === event.target && // Click on ::before,::after pseudo-elements ([iconStart] / [iconEnd])
+            !this.input?.nativeElement.matches(':read-only')
```

Comment:
Should we augment `TuiTextfieldAccessor` with `readonly`-property ? 🤔 

https://github.com/taiga-family/taiga-ui/blob/11c644f98f410a9457729e4619c2bf9e9768fa9d/projects/core/components/textfield/textfield-accessor.ts#L4-L6

Reply:
I don't think we need to expand accessor. A concerns to think about:

We are setting open to true regardless of whether dropdown is there or not, might get us into a sync issue if dropdown content appears later but the open status is `true` and we cannot tell it to show the dropdown because it already thinks it's showing it.